### PR TITLE
Added Dijkstra Algorithm

### PR DESCRIPTION
**Pull request by gunjan543**
Add Dijkstra Algorithm #9

### CODE REVIEW

File: Graph Theory/Dijkstra Algorithm/Dijkstra.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+int main(){
+	Graph<int>g;
+	g.addEdge(1,2,1);
+	g.addEdge(1,3,4);
+	g.addEdge(2,3,1);
+	g.addEdge(3,4,2);
+	g.addEdge(1,4,7);
+	g.printAdj();
+	g.dijsktraSSSP(1);
```
Comment (ekdnam) [inline Graph Theory/Dijkstra Algorithm/Dijkstra.cpp]: Can all these inputs be taken from the user, rather than statically?

**Review by ekdnam — CHANGES_REQUESTED** [review]

File: Graph Theory/Dijkstra Algorithm/Dijkstra.cpp (hunk 1 above)
Comment (gunjan543) [reply Graph Theory/Dijkstra Algorithm/Dijkstra.cpp]: Yes sure!  I will make changes and Update it asap.

**Review by ekdnam — APPROVED** [review]

**Comment by ekdnam** [comment]
LGTM! Thanks for the contribution! Merging into master

**Comment by gunjan543** [comment]
> LGTM! Thanks for the contribution! Merging into master

Thanks for merging my PR :)

**Pull request merged**